diff -ur fonty-1.0.orig/Makefile fonty-1.0/Makefile
--- fonty-1.0.orig/Makefile	1998-09-19 22:52:55.000000000 +0200
+++ fonty-1.0/Makefile	2004-08-18 17:45:04.627513408 +0200
@@ -19,7 +19,7 @@
 consoletransdir = $(consoleprefix)/share/consoletrans
 konwertprefix = /usr/local
 konwertsharedir = $(konwertprefix)/share/konwert
-konwertlibdir = $(konwertprefix)/lib/konwert
+konwertlibdir = $(konwertprefix)/lib64/konwert
 konwertbindir = $(konwertprefix)/bin
 perl := $(shell which perl)
 docprefix = /usr/local
